app/assets/stylesheets/bootstrap_bundle.scss

```diff
@@ -214,6 +214,9 @@ body > div > .popover {
     @import "bootstrap-10";
   }
   @import 'bootstrap-inat';
+  [dir="rtl"] & {
+    @import "rtl-bootstrap";
+  }
```

Comment:
Also, assuming this is the way to go, this needs to be imported beofre bootstrap-inat, otherwise all the overrides in `bootstrap-inat` don't work (you can see this in the `btn-inat` button border radius).